Kernel/Process.cpp

```diff
@@ -2726,6 +2727,28 @@ int Process::sys$reboot()
     return ESUCCESS;
 }
 
+int Process::sys$mount(int devmajor, int devminor, const char* mountpoint)
+{
+    auto* dev = VFS::the().get_device(devmajor, devminor);
+    auto* disk_device = static_cast<DiskDevice*>(dev);
+
+    dbgprintf("mount: attempting to mount %d,%d to %s...\n", devmajor, devminor, mountpoint);
+
+    if(dev == nullptr)
+        return -ENODEV;
+
+    // We currently only support ext2. Sorry :^)
+    auto ext2fs = Ext2FS::create(*disk_device);
+    if (!ext2fs->initialize())
+    {
+        dbgprintf("mount: failed to create ext2fs!\n");
+        return -ENODEV; // Hmmm, this doesn't seem quite right....
```

Comment:
I'm not sure what the right error code would be.. `ENODEV` seems fine for now.

Reply:
Yeah I was thinking the same thing.. It's the only `ERRNO` that I could really think of/see that would make sense in this context. Do you know what Linux returns in this situation??